core/SymbolRef.h

```diff
@@ -381,61 +381,61 @@ class Symbols {
         return SymbolRef(nullptr, 78);
     }
 
-    static SymbolRef Enumerator() {
+    static SymbolRef T_Sig_WithoutRuntimeSingleton() {
```

Comment:
Note: I moved this symbol up to be right after the non-singleton, which caused the rest of the diffs in this file.